roles/tunnel-server/templates/site-https.conf.j2

```diff
@@ -90,7 +93,7 @@ server {
 
     {% for user in tunnel_users %}
     if ($user ~* "{{ user.name }}") {
-      set  $tunnel_port  {{ user.unique_port }};
+      set  $tunnel_port  {{ service.base_port + loop.index }};
```

Comment:
Each user's port is determined by the order of users automatically. That means new users no longer have to pick a unique port, it's just generated for them. 

If this is ever updated, the `/users.json` will also get updated and the client will still work correctly.